src/sentry/lang/native/symbolicator.py

```diff
@@ -318,6 +324,7 @@ def _request(self, method, path, **kwargs):
 
         # required for load balancing
         kwargs.setdefault('headers', {})['x-sentry-project-id'] = self.project_id
+        kwargs.setdefault('headers', {})['x-sentry-event-id'] = self.event_id
```

Comment:
@tonyo Please be aware of this change. Essentially, we would like to move from per-project bucketing to per-event bucketing.

Reply:
Discussed in person now

Reply:
✔️